vite.config.ts

```diff
@@ -15,32 +15,78 @@ export default defineConfig({
       injectData: {},
     }),
     VitePWA({
-      // strategies: "injectManifest",
-      // srcDir: "src",
-      // filename: "service-worker.js",
-      selfDestroying: true,
+      devOptions: {
+        enabled: true,
+        type: "module",
+        navigateFallback: "offline.html",
+      },
+      srcDir: "src",
+      strategies: "injectManifest",
+      includeManifestIcons: false,
+      includeAssets: [
+        "world/*",
+        "pwa/**/*",
+        "farms/*",
+        "offline/*",
+        "offline.html",
+      ],
+      injectManifest: {
+        maximumFileSizeToCacheInBytes: 15000000,
+        globPatterns: ["**/*.{js,css,html}", "assets/*.{jpg,mp3,svg,gif,png}"],
+      },
+      filename: "sw.ts",
       manifest: {
+        name: "Sunflower Land",
+        id: "com.sunflower-land",
+        description:
+          "Plant, Chop, Mine, Craft & Collect at Sunflower Land. The MetaVerse game with endless resources.",
         short_name: "Sunflower Land",
+        start_url: "/pwa/",
```

Comment:
NOTE: NEEDS TO BE CHANGED BEFORE MERGE
```suggestion
        start_url: "/",
```